scaletest/terraform/gcp_db.tf

```diff
@@ -0,0 +1,62 @@
+data "google_compute_network" "default" {
+  project = var.project_id
+  name    = "default"
+}
+
+data "google_compute_global_address" "sql_peering" {
+  name = "sql-ip-address"
+}
```

Comment:
What is this used for?  I don't see any references to it

Reply:
Nice spot, this is stale copy-pasta. Removed.